mod-pubsub-server/src/main/java/org/folio/services/impl/SecurityManagerImpl.java

```diff
@@ -55,6 +54,7 @@ public class SecurityManagerImpl implements SecurityManager {
   private static final List<String> PERMISSIONS = readPermissionsFromResource(PERMISSIONS_FILE_PATH);
   private static final String ACCESS_TOKEN_NAME = "folioAccessToken";
   private static final String REFRESH_TOKEN_NAME = "folioRefreshToken";
+  public static final String SYSTEM_USER_TYPE = "system";
```

Comment:
I think it makes sense to declare this constant as private and create a separate one for tests

Reply:
Made change, thanks